profile/src/v100/factors/factor_instance/private_hierarchical_deterministic_factor_instance.rs

```diff
@@ -0,0 +1,119 @@
+use crate::prelude::*;
+
+/// An ephemeral (never persisted) HD FactorInstance which contains
+/// the private key, with the ID of its creating FactorSource.
+#[derive(Debug, PartialEq, Eq)]
+pub struct PrivateHierarchicalDeterministicFactorInstance {
```

Comment:
Never serialized and need not be UniFFI exported, only ephemerally used internally within Sargon.